ephios/settings.py

```diff
@@ -344,7 +344,7 @@ def GET_SITE_URL():
 # bootstrap v5 or v6. See https://github.com/twbs/bootstrap/issues/25394 for details on the problem and
 # https://security.stackexchange.com/a/167244 on why allowing data: is considered okay
 CSP_IMG_SRC = ("'self'", "data:")
-CSP_STYLE_SRC = ("'self'",)
+CSP_STYLE_SRC = ("'self'", "'unsafe-inline'", "'unsafe-hashes'", "'nonce-rAnd0m'")
```

Comment:
is there any way we can avoid this? we really don't want that

Reply:
Yep, one of three options
* generate a nonce for the inline css
* generate a hash for the inline css
* move the inline css to a dynamic css file

I just couldn't be bothered to correctly implement CSP before being able to write my first line of CSS. Out of these, options 1 and 2 would probably be easiest. Django-csp seems to have some support https://django-csp.readthedocs.io/en/latest/nonce.html#django-template-tag-jinja-extension

Reply:
I've implemented the nonce solution.